src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -24,17 +24,17 @@ export const EditProblemView = ({
 
   return (
     <EditorContainer getContent={getContent}>
-      <div className="editProblemView d-flex flex-row flex-nowrap justify-content-end">
-        <span className="flex-grow-1">
-          {isAdvancedProblemType ? (
-            <RawEditor editorRef={editorRef} lang="xml" content={problemState.rawOLX} />
-          ) : (
-            <>
-              <QuestionWidget />
-              <AnswerWidget problemType={problemType} />
-            </>
-          )}
-        </span>
+      <div className="editProblemView d-flex flex-row flex-nowrap justify-content-around">
```

Comment:
I cannot say for sure, but I would say adding space around the content could cause problems (if the content is not big enough).